config-chrome.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
 # script to run the sed commands in both OSX and linux (e.g., GitHub Action)
-# replace "menus" permission (Firefox) with Chrome-compability permission ("contextMenus")
+# remove "contextMenus" permission (Firefox-only) from manifest.json
```

Comment:
This code is removing "menus" from manifest.json when I do `npm run web-ext-run:chrome` and whenever the scripts are run in github actions. Removing "menus" would cause the firefox context menu to not work, unless I manually add it back. 

Is there a reason for this and is this intended? Should we have a config.sh for firefox seperately that removes contextMenus and re-adds menus when running the firefox extension?

In the builds from github actions and the one tested in the original bug report https://github.com/mozilla/fx-private-relay-add-on/actions/runs/6844016974 ([MPP-3581](https://mozilla-hub.atlassian.net/browse/MPP-3581)), if you download the zip file you can see that only "contextMenus" is in the manifest.json. This is might be the reason why QA ran into the bug, since Firefox needs "menus". 

Is there a reason why we can't leave both menus and contextMenus in the manifest.json file?

Also confirming that the context menu works, but only when "menus" is in the manifest.json file.

Reply:
Ahh! I should've flagged this change better in the self-review / code comments. 

The permission, [`menus`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/menus) does not exist for Chrome. Previously (before Oct 2023), we could ship permissions that Chrome did not recognize, check if they exist, and use them as available. However, Chrome rejected our submission and needed a clean (read: no declared `menus` permission) manifest file to approve our release. 

When we did this change, we re-submitted and Chrome accepted, etc. (and special note that we did NOT submit a Firefox release with this new change) 

QA flagged the following issue on the Firefox side: https://mozilla-hub.atlassian.net/browse/MPP-3581 where if you didn't ship BOTH permissions to Firefox, _it_ would break. 

That put us in a pickle. Either refactored a bunch of the code everywhere `.menus` and `.contextMenus` is mentioned OR ship both permissions to Firefox and only `contextMenus` to Chrome. 

This change to both the `config-chrome.sh` file and the `manifest.json` file (where both permissions are now included) makes the add-on work across platform.